src/MDMPerforming.h

```diff
@@ -148,6 +148,30 @@ NS_SWIFT_NAME(ComposablePerforming)
 
 @end
 
+/** Specifics for a named plan performer to allow named plans to be added and removed. */
+NS_SWIFT_NAME(NamedPlanPerforming)
+@protocol MDMNamedPlanPerforming <MDMPerforming>
+
+/**
+ Provides the performer with an plan and a name.
+
+ @param plan The plan that required this type of performer.
+ @param name The name by which the plan can be identified.
+ */
+- (void)addPlan:(nonnull id<MDMPlan>)plan
+       withName:(nonnull NSString *)name
+  NS_SWIFT_NAME(add(plan:withName:));
```

Comment:
Hmmmm. Perhaps the `removePlan` function on `MDMPerforming` should be more like `removePlan:withName:` per https://material-motion.gitbooks.io/material-motion-starmap/content/specifications/runtime/named-plans.html.